socorro/tests/external/es/test_crashstorage.py

```diff
@@ -94,10 +95,10 @@ def test_invalid_key(self, key):
 
 class TestESCrashStorage:
```

Comment:
`ELASTICSEARCH_MODE` is also set in `bin/test.sh` for tests, to ensure we test both modes.

i did it this way to match what I did for gcp vs aws during the gcp migration to ensure that tests run and fail in CI instead of silently skipping if configuration wasn't working as expected. 

that said i'm fine switching to `pytest.mark.skipif`, just confirm for me which you'd prefer